astropy/coordinates/tests/test_frames.py

```diff
@@ -180,7 +184,9 @@ def test_frame_repr():
         assert repr(i3) == ('<ICRS Coordinate: (ra, dec, distance) in (deg, deg, kpc)\n'
                             '    [(1.1, -15.6, 11.0), (2.1, 17.1, 21.0)]>')
 
-
+@pytest.mark.xfail(WINDOWS and six.PY3,
+                   reason='Python 3.x on Windows rounds different from '
+                          'everything else for unknown reasons')
```

Comment:
I'm utterly baffled right now.  Because you had previously asked me to investigate the failure of this test on Windows. And I found that it was not WIndows-specific at all (in fact this test fails for me on Linux), and I even submitted a patch for the issue.  Now for the life of me I can't find any of that....